packages/block-editor/src/components/plain-text/content.scss

```diff
@@ -1,11 +1,27 @@
-.block-editor-plain-text {
+.block-library-html__edit .block-editor-plain-text.components-base-control {
+	padding: 0 !important;
+}
+
+.block-editor-plain-text textarea {
 	box-shadow: none;
 	font-family: inherit;
 	font-size: inherit;
 	color: inherit;
 	line-height: inherit;
 	border: none;
-	padding: 0;
+	padding: 12px;
 	margin: 0;
 	width: 100%;
+	max-height: 248px; // 250, minus the top and bottom border (1px each)
```

Comment:
The wrapper has `250px` max-height declared somewhere else, so we're adapting the `textarea` element to match it.